flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/strategy/PipelinedRegionSchedulingStrategyTest.java

```diff
@@ -174,10 +174,12 @@ public void testFinishedBlockingResultPartitionProducerDoNotScheduleNonCreatedRe
 
         final PipelinedRegionSchedulingStrategy schedulingStrategy = startScheduling(topology);
 
+        // non-CREATED regions should not be re-scheduled
         consumer.get(0).setState(ExecutionState.SCHEDULED);
```

Comment:
now we do not expect this case to happen.
exception should be thrown in this case.

Reply:
I'll remove this case, since the changes of FLINK-21707 should be removed along with this pull request.